### PR TITLE
Makes Gateway Explorer an official job

### PR DESCRIPTION
**Pull request by Kyep**
Reasons why we should have Gateway Explorer as a job:
1) We need more job slots. With >100 people regularly on the server, adding new jobs is important in order to avoid having 50+ Civilians on shift. Gateway Explorer, as a potential job, is a popular job, with existing mechanics, that can support several slots, and requires almost no map edits to make it viable (just placing the starting positions). In terms of jobs we could add to boost slots, it does well on every measure. If we want to add new jobs to increase slots (which we have to, really), then Gateway Explorer should be at the top of the list.
2) It is already an unofficial job. We get several people asking for it every shift. This isn't good, because, as an unofficial job, its not managed or regulated very well. It doesn't show up on sec HUDs, it requires a lot of time from the HoP to just process 'I want to become a GE' requests, etc. We don't accept "oh, lets leave this job out of the code and make people ask for it manually every shift" for ANY other job on the station that people regularly do. Why should we accept it for Gateway Explorer? We shouldn't. It should be an official job like any other.
3) GE provides an outlet for people who would otherwise Greytide. Now, they can go off into the gateway and kill things / destroy stuff there. Fun for them, less hassle for everyone else. We have the pop to support it.

OK, so, you've convinced me. What do Gateway Explorers get?
- Starting position in Gateway room, with Gateway access.
- That's it. They have to scrounge all their supplies by ordering them from cargo, interacting with / persuading the crew, etc.

Example screenshot of a GE start:

The green jumpsuits are standard-issue. The white/gray 'G' that resembles a swirling whirlpool is their sec hud icon.

This was brought up on the forums recently here: https://nanotrasen.se/phpBB3/viewtopic.php?f=12&t=9907
While the forum thread calls for making them a Karma job, personally I think they'd do MUCH better as a regular job. So, I did not make them a karma job - they're available to everyone.

🆑 Kyep
add: After a long history of station crews sending out Explorer teams, NT has recognized this as a job.
/🆑

### CODE REVIEW

**Comment by Jovaniph** [comment]
It would need an SoP as well...also it should be 100 karma (of whatever)

**Comment by Aurorablade** [comment]
if someone wants to greytide they will greytide, Regardless. Plus theres alot of unoffical jobs the hop can give out.

I am not sold on this.

**Comment by LightFire53** [comment]
I would say add two eva suits to the gateway then, and leave regular eva untouched. Although, regular eva doesn't really have a purpose...

**Comment by Aurorablade** [comment]
that and i am gonna add we don't really have that much of exicting gateways besides wild west...so..RNG happens..you can now greytide in the gateway...on the beach.

**Comment by taukausanake** [comment]
These should be a karma job I feel. Gateway explorers don't really add anything to the round; the very most they can get loot that will end up in Science to get researched or abused if possible. Also that EVA access shouldn't be free.
Keep the space suits in EVA and let them have access. You want to keep the suits together else people will just break into the gateway and take those suits instead.
I think I'd like them to have their own jumpsuit or at least uniform. Something a little standard, like they are trained professionals (or chuckle fucks) at exploring exotic and hostile lands. That or give them red jumpsuits. Never to be seen again :P

**Comment by SamHPurp** [comment]
I'm really against holding this as a karma only role...

I agree with pretty much the only PR, I'm enjoying the fact that nobody is complaining about them not having maintenance access, personally I think thats genius.

One thing I'd push for changing @Kyep is indeed as you nodded to... removing their EVA access, and adding a couple soft suits to the Gateway room.

Other misc concerns: "Boring" gateway missions like The Beach, or the one that requires you to dig out of it.

**Comment by taukausanake** [comment]
I agree with no maint access and I suppose if you can have air tight soft suits (I'm not aware of any so please correct me) that might be better than EVA access.

There is one thing I'd like to mention now that I recall it but not of importance. A gateway team is a group comprised of members from different departments, kind of like an ERT squad. It's more IC stuff though but I think it would be nice if they did have a job jumpsuit that they have armbands for which sect they should be in. Just a little bit more for immersions.
Otherwise, the only downside I see to them having red shirts is that they might look like Security too much but spriting isn't that hard

**Comment by Rumiluntti** [comment]
I am against making more jobs that encourage you to leave the station.

**Comment by SamHPurp** [comment]
@Rumiluntti Have you been HoP lately? You'll get between 5-15 people asking you to be a Gateway explorer per shift.

This is already treated as a job by the entire population. This PR removes that artificial barrier.

**Comment by Xantholne** [comment]
https://youtu.be/MnFQlVT2UBo

**Comment by SheepFighter** [comment]
My only complaint is the lack of maintenance access. Is there a single role currently that doesn't have it? 
Not that I'm necessarily opposed to restricting maint access, but let's at least be consistent about it. 

Otherwise, this looks good.

As for the idea of this being karma-restricted: That does seem to defeat the point of this PR, i.e. stemming the greytide. Besides, the away missions at present are no great shucks. They'd need to be renovated greatly to be merit anyone spending kama on them.

**Comment by jughu** [comment]
Neat a powergame job its likely not gonna end in the following

Xenomorph's
Powercreep man validhunting on return

**Comment by Kyep** [comment]
@LightFire53 @SamHPurp Removed their EVA access. Put two suit storage units in the Gateway room to compensate. This prevents them from giving clowns/mimes their EVA suits with no oversight.

@SheepFighter @jughu The fact that they lack maint access prevents them from becoming simply a better version of assistant, and also reduces their ability to validhunt after returning from the gateway.

**Comment by jughu** [comment]
What would them stop from walking to hop and ask maint access?

**Comment by SheepFighter** [comment]
@Kyep Now _that_ makes sense. I thus have no complaints.

**Comment by Kyep** [comment]
@jughu Nothing. The HoP probably won't, though, unless the Gateway is off that shift.

**Comment by jughu** [comment]
This could still cause peoplw releasing xenomorphs on purpose by facehugging themselves

**Comment by jughu** [comment]
Also im not against the idea of having a job about it i just feel the maps should be more balanced before we get this and maybe with a telescience gimmick to it

**Comment by Kyep** [comment]
@jughu There is no way to get xeno infected in the Gateway. All xeno eggs/larva were removed in previous PRs. Additionally, we have people taking unofficial gateway explorer jobs every shift already. This merely codifies an existing job. Telescience is hated by many, to the point it was removed from Science (there used to be a telesci console at roundstart). I won't add any form of telescience to this, as I know it would be unpopular, and this PR exists only to codify what is already popular / common practice.

**Comment by davidchan13** [comment]
I'd make them their own unofficial dept and make an away team leader as the subhead (like quartermaster) with the leader as the karma role and the regular team members either being timed or cheap roles (like barber) as you could in theory have a 5 man team of leader, Doctor, scientist, engineer and security. Each of them spawns with a soft suit in their bag/internals and VIP space tourist slots could be added if the roles pick up and more missions come up. 

Otherwise they could always go explore the derelict or mining asteroid

**Comment by Kyep** [comment]
@davidchan13 I don't want any karma locking in this PR. It complicates the PR significantly from a technical standpoint, it means people who used to lead away teams no longer could unless they paid karma to do so, and it introduces additional balance/gameplay problems. I'd be fine with colored armbands pre-spawned in the gateway room, perhaps in a locker, so people can put on the bands to signify they're taking certain roles within the team. I don't want to get really specific about the roles within the team, though. People can already self-identify if they want. Softsuits don't fit in bags, but the team already gets expanded internals in their internals box. "Space Tourist" might be a thing to consider in future, perhaps as an alt title for explorer. Right now, though, I don't think its needed. You're right that these Explorers could also explore the derelict, if the mechanic builds them a pod or two. Or they're willing to drift about in space until they find it.

**Comment by Draconic-saint** [comment]
Gateway right now isnt suited to be a job contentwise, and this will do little to alliviate the tide. Not sold

**Comment by uraniummeltdown** [comment]
Agreed with Draconic, the only away mission that fits a team of explorers is the Wild West which people usually complete by preparing for beforehand

**Comment by Fox-McCloud** [comment]
Not sold on this, at all.

For one, everyone is meant to have maintenance access by default, right now. While this is subject to change, in the future, not giving them maintenance access is a huge inconsistency.

That said, even with that change, I'm still opposed to this, for a few reasons. 

It encourages people to be off station instead of being on-station and doing their job, it doesn't really have an end goal aside from validhunting or other similar purposes, it's a greytide+ role, far worse than barber (they get a free space suit, near-full toolbelt, and healing patch).

There's going to be a significant portion of players who don't actually do gateway exploration stuff with this and, instead, faff about on station or even go deep space exploring.

Is it a new job? Yes. Will it reduce greytide? Maybe a little, a best, but it's not a particularly well thought out job, and discourages co-operative gameplay and generally encourages behaviors we don't want to.

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -26,3 +26,37 @@
...
+	department_flag = SUPPORT
+	total_positions = 5
+	spawn_positions = 2
+	supervisors = "the head of personnel"
+	selection_color = "#dddddd"
+	idtype = /obj/item/weapon/card/id
+	access = list(access_gateway)
+	minimal_access = list(access_gateway)
+
+	equip(var/mob/living/carbon/human/H)
```
Comment (tigercat2000) [inline code/game/jobs/job/civilian.dm]: Relative pathing.

**Comment by SamHPurp** [comment]
I straight up cannot agree with any of your points @Fox-McCloud .

This is already a job, as treated by the entire population. Nobody new is being 'encouraged to leave the station' more than they already are.

The maintenance access restriction is a move to reduce it being a greytide+ role as you so fear. Also with that access restriction it helps tone down them coming back and being a validhunter (So that even further improves on the current situation).

As per them exploring deep space instead, well thats still also what people do currently if the gateway mission isn't interesting.

The one point I will entirely concede is that all of the away missions, other than wild wild west, is dull and easy to complete.

**Comment by Draconic-saint** [comment]
Now that I have some free time on my hands i'll write a proper comment instead of a fast mobile twitter-style one.

First off, clearly it's not regarded as a job by everyone either, since this discussion is taking place. 

And please stop saying it's already "de facto" a job. It's not. There's no SoP, no objective or purpose. It's as much a job as civilian is. It's a lack of a job on, & for the benifit off, the station, you're in the round but you couldn't be arsed to be useful or to interact with the crew.

I don't want to encourage that. And by turning gateway explorer into a job that is exactly what you will encourage. More people fucking off and doing their own thing instead of contributing to a game that at it's CORE is socially driven by interaction on the station. 

As for removing the maintenance acces, that invites inconsistancy, and makes no sense from many perspectives. It's litterally a sloppy patch because your core idea is flawed and you don't want to face it.

And it's not even a good patch, because the toolbelt is easily used to circumvent that restriction. They even get some meds to fix any screwup they make, a luxery the ordinary greytider doesn't get. 

This PR will only invite more tide, more self-centered play, and more unwanted behaviour in my opinion.

That adresses some of the points the propennents made for this PR, let me add to this that there is not nearly enough viable gateway content for this to be even considered having any replayability.
What're you going to do, hope it's wild west every single time? 

A great deal of the effort towards the gateway now goes the fact that you need to get a squad & equipment together, and prepare. And that's great, you get to interact with different departements, talk with different people. You'd be cutting that very neat RP experience out because some folks are too goddamn lazy to go to the HoP's. It's an experience I think every veteran player on this server has had. Because we don't have gateway explorer as a job.

To summarise: It's not fit to be a job, it can only increase the tide and the removed maintenance acces is a bad fix, there's not nearly enough content, and you'd be killing a very neat RP scenario that occurs because of the absence of the job.

Please don't make this a job.

**Comment by SheepFighter** [comment]
@Draconic-saint 
"[B]y turning gateway explorer into a job that is exactly what you will encourage. More people fucking off and doing their own thing instead of contributing to a game that at it's CORE is socially driven by interaction on the station."
How would this be any different from miners? Space Station 13 contains multitudes, and while much of it is driven by interaction between players, it's eminently accommodating of outliers. This is the beauty of it, IMO: There's something for everyone. Want to roleplay and nothing else? Great, you can do that. Want a mix? Ditto. Want a more gamey experience, away from the madding crowd? Again, you can get that.

More pertinently, Paradise is so high-pop that this wouldn't make a dent in the gameplay. And it's being proposed as a means of stemming precisely what you fear: People who don't contribute or interact in anyway beyond powergaming and breaking things. Greytide at present is quite an issue. This would hopefully redirect it off the station, if nothing else.

"A great deal of the effort towards the gateway now goes the fact that you need to get a squad & equipment together, and prepare. And that's great, you get to interact with different departements, talk with different people. You'd be cutting that very neat RP experience out because some folks are too goddamn lazy to go to the HoP's. It's an experience I think every veteran player on this server has had. Because we don't have gateway explorer as a job."
In my experience, gateway prep is never this detailed. I go to the HOP, acquire a toolbox and a spacesuit and a station-bounced radio, and that's it. The only scenario that requires any prep is the wild west one. 

Nor is there any reason that making Gateway explorer a job need cut out preparation. You could start with the bare minimum, then gather supplies and recruit additional explorers (were that actually necessary.)

I think the one thing we can all agree upon is that the gateway missions are severely lacking. For my money, the only worthwhile ones are the space hotel and the wild west. The beach works for what it is, but what it is is rather paltry. For this job to be worthwhile in the longterm, we'll need a major overhaul of the away missions. Perhaps it would be best to leave this PR until that happens, so we have a better idea of what we're getting into. On the other hand, merging this would make an overhaul more imperative.

**Comment by Kyep** [comment]
@Fox You're not worried that Gateway Explorers would be Assistant+ if they had maint access? I wanted to give them a con, and that's what it was. I'm unsure if it really encourages people to go off station significantly more, given how there are unofficial GEs almost every shift already. I'm not even sure getting some of the civilians something to do, even if off-station, is a bad idea. We've had way more radical things (second server, second station zlevel on same server with different crew, etc) proposed as means to deal with highpop. This is WAY less radical than any of those, and could give several people per round something to do. With this PR, official GEs will be at a disadvantage when faffing around on the station itself, as they lack maint access. Someone who wants to faff around on the station should choose Civ, not this. If they decide to go into space exploring, though, I think that's a good thing. They'd do it anyway - might as well recognize it. I don't think this job discourages co-operative gameplay - if anything it encourages it, as gateway teams typically have to work together to perform at their best. 

@tigercat2000 Fixed.

@Draconic-saint Actually, there already is SoP for the Gateway. It will require update with this, but it does exist. Specifically it will require changing from "Captain must authorize all missions" to "Captain can veto any/all missions, and is encouraged to do so on blue and higher alert levels" or similar. I doubt GEs will stay in the Gateway all shift. Typically they don't right now. Yeah, they take a trip, but then they come back. Turning GE into an actual job won't make them stay there longer than they do currently, it will just recognize what they are already doing. I do agree with you that we need more interesting Gateway content, and I'll be putting up another PR soon that adds a new Gateway map. This PR doesn't give GEs everything they could need. They'll still need to collect supplies from around the station for the more challenging missions. Even then, I could easily not give them a toolbelt/medpatch/flashlight, forcing them to get these things from Cargo instead. That would drive interaction and make them make prepping choices, whilst still recognizing the job.

**Comment by Draconic-saint** [comment]
@SheepFighter ( And by extent @Kyep since I'm not gonna adress some issues twice)

1) It's massively different from mining, because mining has a reason to exist: bring back materials to the station for repairs/research and improvements. It requires you to still be a part of the station, although to the very minimum.

This does not. Right now there's no goal, no sop beside the **single line** that Kyet pointed out. No guidelines on what to do with found objects, no guidelines on what do to with explorers who died, nothing.

2) It the past there used to be more lethal gateway expeditions that actually required plenty of gear, like the emitter maze, or the Santa fight during Christmas times. True maybe my view is skewed because I also experienced these. But even acquiring the tools, the simple med-kit, the spacesuit, that's still a solid 15 minutes you're on the station, interacting with people and gathering stuff. 

3) Both you and others claim this would redirect greytide off the station. It's a temporary spongebob themed band-aid at best on a gushing cut. Unless it's wild west, I don't see people in the gateway for longer than 40 minutes. So that's still about an hour they will be spending on station, counting a generous 20 minutes prep time. So now you got bored explorers back on the station with tools and a spacesuit on hands. Potential for greytide+ in my eyes.

I'm not oppossed to thinking how this may become a full-time job. I had very similar ideas for xenoarch. But this isn't thought out at all, I feel that it's a rushed kneejerk PR because of vocal people that want it as a job. My advice would be to close this PR, add more missions, missions with substance to them. Then think about how it could fit as a real job into the station. Think about the synergy between station and gateway explorers. Because right now, there's very little besides showing off what you found. And then having it confiscated by sec.

**Comment by Kyep** [comment]
@Draconic-saint Mining doesn't really have an endgoal outside of 'acquire better mining gear, rack up a bajillion points'. Yet, that is a job. The things they bring back can be useful. It isn't a high bar to pass. If you're arguing for more guidelines for gateway explorers, that points in the direction of formalizing the job MORE, not less. I am going to add one fully-developed, dangerous gateway mission in the next two days or so, and I have an idea for another beyond that. As I said, it would be easy for me to simply give them less gear at roundstart, so they have to spend more time gathering stuff. I don't see how having a bored explorer on station for an hour is worse than having a bored civilian on station for two hours. This PR came out of the 'we need more job slots' considerations in the OP. It was not based on any vocal person asking for it. The fact it has proven popular is just a bonus. I wouldn't be opposed to reviewing our gateway missions with an eye to improving them. But, the fact is, most shifts there are at least a couple of gateway explorers right now. If they find it a valuable enough job to do, right now, even without more gateway missions, why not let them? Formalizing the job that they're already doing makes managing the gateway teams easier.

**Comment by jughu** [comment]
Theres not enough gateway content to make it a job like
The most likely maps are
Wild west(death sentence if you dont get weaponry plus additional weaponry hardsuits eshields eswords)
The hotel thing with the snpc officers with tasers(free tasers)
Beach(basically they enter the gateway directly leave because it isnt fun and just rp fluff)
Xeno thingy(just a hive with night vision and one stetchkin after fighting 10 aliens and a empress)
Emitter maze(dont people just deconstruct the emitters and go trough the sides nowadays....)

If they was more mechanics to the gateway like buying coordinates for certain missions with cargo budget and then calibrate into these positions

And more maps be surely needed not 100% loot based gateaway missions are fun of full rp with a little bit of loot mixed to it(most likely loot that will help you progress on the mission so you can bring it back and  your friends!

But for now its not really needed contentwise

**Comment by Draconic-saint** [comment]
On mobile but

Mining has synergy with science, cargo and engineering, and other departements have use for the things they bring back. Supplying the station with metals en resources is the goal.

The points are a reward, they shouldn't be considered the goal of the job.

& Just because people do it every round doesnt make it a viable job. Should fort builder be a job? Should holodeck fighter be a job?

Right now, the gateway is a side activity. Get ot the content, synergy & guidelines first, now you run the real risk of making it a job and then leaving it half finished for a loooong time because the other changes might not get merged immediatly.

The reason i dont like this as a tide solution is that you're not adressing the issue, you're hiding it. You give them a toybox they CAN wreck but once thats over the same behaviour is coming back into the main station.

**Comment by taukausanake** [comment]
I do want to add to this that pod engineer is a job and that is literally "make a space pod so you can go bugger off in space away from the station". Space Explorer the job basically and I kind of enjoy it. That's my reasoning for this being a karma job as it doesn't add anything to the station

**Comment by Kyep** [comment]
## Major changes have been made:
- GEs no longer get toolbelts, fingerless gloves, a styptic patch, a seclite, engineer emergency tank, handheld radio, or EVA suits in the Gateway room. This forces them to scrounge for supplies on the station, interacting with the crew, for an extended period of time before they leave. They risk quick death otherwise.
- GEs now get maint access, because Fox wanted all jobs to have maint access.

## Much of the community is in favor of this PR.

On Github, the reactions are currently 12 positive, only 3 negative. That's four times as many people like it as dislike it.

Just to be check whether this reflected broader community opinion or not, I did a server poll just now. This was the poll:

Do you think that Gateway Explorer should be made into an official job?
Yes: 43 votes
No: 5 votes

43 votes out of 47 is >91% support for the idea. 91% support.
I realize the people who are against this may have their reasons, but to put it bluntly, they appear to be a minority. Public opinion seems to be very much in favor of making GE an official job.

**Comment by jughu** [comment]
Because people want it doesnt always mean they get it

**Comment by Draconic-saint** [comment]
Polls on the server are yes/no without much context. I could easily do my own poll in the line of "do you think there are enough gateway missions right now?" and it wouldn't surprise me to see a decent shift into the "no" territory. I put zero faith in that. I've always put zero faith in polls where people do little explaining of *why* they voted a certain way.

Public opinion just means little, because in general the public is uninformed. I'm willing to bet money that most didn't think further than: "Gateway can be fun, I'd like to do it, let's make it job."

It's not that simple. And this isn't a direct democracy.
If that's good or bad doesn't have a place here, but it's the maintainers that decide if this get's merged, not some yes/no poll.

You've already gutted your PR, hoping to reach a compromise in a situation where there is no room for compromise. There just isn't enough to it at the moment for me to consider this a worthwhile addition. the content right now is sufficient to be a side-activity, like building a fort in the tunnels, or starting a shop in the abandoned office. Not a full-time job.

If you'd make this PR again after making sure this job has a decent FOUNDATION to rest upon, instead of 5-6 random gateway missions, I'll support it. But not right now.

**Comment by Kyep** [comment]
There are many jobs, right now, that have less content than Gateway Explorer has.
To give just a few examples: atmos techs are almost entirely redundant under LINDA, psychiatrists aren't fleshed out at all, and librarian is close to assistant with a custom title and an office.
Yet, all those are jobs, and Gateway Explorer... isn't? That makes no sense. Gateway Explorer has more content for it, as a job, than ANY of those jobs. It deserves to be a job more than they do.

A job doesn't need to be perfect to be a valid job. There are plenty of jobs with VERY little to do. Such as the above. There are also jobs with almost no crew interaction, like virology, and some shifts, xenobiology. Heck, those are even annoying in many of the interactions they do have, releasing annoying viruses or tame xenos. Yet, they're considered important jobs. Why?

Gateway Explorer surpasses the standards for job-worthiness set by those other jobs. If they deserve to be jobs, then so does Gateway Explorer.

**Comment by jughu** [comment]
Unless theres a real mechanic added to gateway with reasonable content it shouldnt even be a job since you can litteraly throw a video camera in to see which mission it is so 👎 untill then man get first on the content/mechanics then back to the job maybe

**Comment by SamHPurp** [comment]
Compromise is the death of good ideas unfortunately.

**Comment by Draconic-saint** [comment]
The differences between the jobs you just described and your idea are numerous. I feel that comparing jobs to each other is rather moot, since they all are supposed to fill a niche in specific way, but i'll play along:

 - librarian/psychiatrist are (currently) RP based jobs, purely for fluff and because people WANT to RP. Their diversity and replay-ability come from the people around them. Social interaction is at the core of this game, and having a few jobs just be centered around that interaction is not a bad thing.

- Atmos tech's currently are in a shit shape yes. But don't forget they're a remnant from the ZAS era from when they were absolutely VITAL to engineering in case of fuck-ups.  And just because I see a guy in a wheelchair doing fine in life doesn't make me think it'd be alright to end up in a wheelchair. It's a bad measurement unit.

- Xenobiology/virology lack interaction, but the fact that they're based on station, already gives me much better hopes at there being interaction between the PEOPLE that do the job, than i'd hope for GE's. Sure they can be cooped up in their lab's all round, but there's potential for everyone to get involved in their shenanigans. GE's don't have that potential once the leave through the portal. And before you point to miners, they have a comms relay that allows them to at least still contact the station, and regularly drop back in and out when delivering mined resources.

You seem to think that code logic also applies to job design. 
"It's got more of _X_ than A, and more _Y_ than B. So my C is better than either of them!

That's not how it works. Some jobs require more interaction, some require more content. There's no absolute minimal values for either of these aspects, but right now, I feel GE's are lacking severly on content. The compromise allows for more interaction, but there is still not enough hands-on content through the portal, and no solid baseline on why it's a job on the station, other than "people want it".

**Comment by Kharshai** [comment]
I really don't like the idea of this being a job, for a variety of reasons.

This job produces nothing for the station, or contributes to its upkeep - it's barely even a job in itself, as all you're even doing is finding loot for yourself to become powerful with. All I see this being is an alternative to civilian except you don't have to scour maintenance to get loot, it's literally just your job to go to the gateway and get powerful items for yourself, and nothing else.

 If the gateway had more of a purpose, such as having useful science items to find or anomalies then I can see at least some reasoning for it being a job, but as it stands, I see none.

Additionally, there's not even slightly enough content in the gateway for this to be a job. It'll give someone something to do for 10-20 minutes, get some good gear, then go back to the station far more powerful and ready to powergame or go hunt down antagonists, and I don't want that behaviour being encouraged at all.

**Comment by ktoma36** [comment]
Perhaps tying in anomalies with gateway missions would be a good idea. Instead of making an anomaly a random event during a round make them appear in missions. I have always wanted to learn how to use the anomaly machines at the sci outpost. To be honest I would love to have the sci outpost be a vital part of science rather than a occasional antag base of operations. That's for another PR though.

As for the gateway itself perhaps making a console controlling where the gateway goes would be a good idea. Getting the beach/listening outpost three shifts in a row is quite boring as everyone has pointed out. With that added we could have a sort of system of unlocking the more dangerous missions. Perhaps a GE would need to find a set of coordinates to the wild west mission sight from the listening outpost, or a magical orb gives the location of the wizard ship. Something that gives a goal other than loot the place and get out. Maybe add random encounters to some of the missions. For example: "You and your crew are getting drunk at the mini bar alongside the beach. Suddenly you hear metal screaming as a ship crashes into the shallows. You have only seconds to prepare as wounded syndicate operatives pry themselves out of the wreckage." It would add more excitement and reason to stick around a particular mission. 

As for the lack of gateway missions. Maybe making some missions more modular would work. Have assets and area's that are designed to fit with each other, making it feel like more of a rogue-like in terms of difficulty and interest.

Another thought entirely that goes along gateway in general. Perhaps antagonist based around gateway missions would be interesting, kind of like a dungeon master per say. This could add a multitude of scenarios. Using an interface like the blob or xenobiologist a so called "GM" could spice things up. This of course would need a LOT of work to be viable, plus it would most likely be a whitelist role as it would give a player admin like power in a smaller setting.

**Comment by Kyep** [comment]
Added sprites that @Xantholne did. OP screenshot updated.

**Comment by jughu** [comment]
But to which department are there science? Command? Mmm

**Comment by Kyep** [comment]
Currently, they're civilian. They report directly to HoP. I can make them part of a department if people want.

**Comment by taukausanake** [comment]
No, they belong to the HoP. You might be able to make an argument that they belong to the sub-head QuarterMaster (I've always thought of him as one) but that's a stretch. It would be interesting under HoP though because they are technically not Service. Maybe a merge between Science and Service/HoP with HoP more in charge? That could be interesting

**Comment by Necaladun** [comment]
First up - imo this has enough supporters and detractors to mark controversial.

Secondly....

What about starting them with an exile implant?

**Comment by emmanuelbassil** [comment]
The worse round I've ever had on this server is when I was IAA , and the greytide let loose on the gateway.
3 people with gateway access let the whole greytide in, while they were stealing supplies and vending machines from the station and heading down there.
Given that security was busy with the antag of the day, and they they deconstructed the doors the Captain bolted, reinforced walls the engineers had built, etc.; it only ended with them shutting power off to the gateway, setting 5 people as wanted and standing at the gateway with beepsky in hand waiting for them to return.

The gateway is a high-security area, I'd like to remind everyone. One where antags can evade justice, and petty criminals turn into headaches. When I HoP, I never hand out gateway access without the Captain's direct approval, and when I'm Captain I never give out gateway access period.

This will just codify the headache and make it unstoppable.

**Comment by taukausanake** [comment]
Man, that must have been one fun round. I really enjoy what the tide can accomplish sometimes

**Comment by Kyep** [comment]
@Fox @uranium @Draconic @jughu @Necaladun @IcyV @SamHPurp @deep-fried-faceplant @DragonFire101 @SheepFighter @scrubmcnoob @andrii-yukimchak @Dadols @DrunkDwarf @Daliden @taukausanake @Doukhan

Major update...

# GE job slots now default to zero. The Captain/HoP have to increase them in order for people to sign up as GEs. This ensures Captain/HoP retain control of the number of slots.

**Comment by emmanuelbassil** [comment]
That sounds like a good idea.
Best of both worlds.

**Comment by SamHPurp** [comment]
Doesn't do much to solve the huge amount of civilians with this neutered in this way.

Again, compromise is the death of good ideas... but something is better than nothing, I guess?

**Comment by jughu** [comment]
👍 now i can be a lazy hop

**Comment by Draconic-saint** [comment]
Think you pinged some poor sap instead of me. But I feel that this has the opportunity be a good start to get built that foundation i've been so adamant about. This keeps command in full control over gateway traffic, and reduces the hassle of doing so. 

I'm still on the fence about having it be an actual spawn able job, since now we can have HoP unleash a horde of explorers on the beach.

**Comment by Aurorablade** [comment]
"GE job slots now default to zero. The Captain/HoP have to increase them in order for people to sign up as GEs."
If you have to do that, just don't make the job and let the Captain/HoP handle it in the first place.

**Comment by Markolie** [comment]
This also needs to have a preview icon added to preferences_setup.dm.

**Comment by Kyep** [comment]
@Markolie done

**Comment by Lady-Luck** [comment]
I didn't used to be for this, but given the greater population we have nowadays, I think it needs to happen.

**Comment by Kyep** [comment]
@Fox-McCloud Merge conflict resolved. Also, updated to use datumized outfits.

**Comment by Necaladun** [comment]
This will likely be closed soon, as it appears heads/maints are universally against it. Others might pitch in, but my personal reasoning is as follows:

1: It does a lot more to legitimize the gateway. We've always been hesitant on such off station activity that doesn't relate to the main station. Currently it's something we don't have to think too much about - having a job revolve around it would make that more difficult.

2: Graytide + Worries.

3: Lack of particular goal - although each gateway has generally some kinda puzzle, there's nothing obligating them to do that, and nothing to do afterwards. My worries here are them coming back to the station. An exile implant to start with might change that....but seems extreme.

4: Lack of SoP, etc. This kinda goes into the above, but who their CO is, etc, etc, really is a big part of any job like this and needs fleshing out.

**Comment by dapocalypse** [comment]
Honestly GEs should be allowed to leave. The purpose of the gateway IC according to the pamphlet is colonization. The goal of GE should be to survive and thrive.
